packages/apps-config/src/links/dotscanner.ts

```diff
@@ -7,10 +7,11 @@ import { externalLogos } from '../ui/logos';
 
 export default {
   chains: {
-    Polkadot: 'polkadot'
+    Polkadot: 'polkadot',
+    Kusama: 'kusama'
```

Comment:
```suggestion
    Kusama: 'kusama',
    Polkadot: 'polkadot'
```

I'm assuming alphabetical ordering is the linting problem.